src/global/Constants.h

```diff
@@ -208,7 +208,7 @@ inline auto& RuntimeParameters() {
                   " must be strictly positive, was ", value.count(), "s")};
             }
           });
-      return parameter;
+      return AD_FWD(parameter);
```

Comment:
I think the move should be already implicit here as of C++20, but it doesn't harm to write this.

Reply:
(IMHO it shouldn't be necessary, but also doesn't pessimize anything. Did some tool tell you this?